spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -43,99 +44,619 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
   private val datePattern = "0123456789/" + whitespaceChars
   private val timestampPattern = "0123456789/:T" + whitespaceChars
 
-  ignore("cast long to short") {
-    castTest(generateLongs, DataTypes.ShortType)
+  test("all valid cast combinations covered") {
+    val names = testNames
```

Comment:
That is awesome.

Reply:
Thanks. The nice thing is that this will tell us if Spark adds or removes any supported cast operations in the future.